pygfx/renderers/wgpu/_shader.py

```diff
@@ -1,15 +1,25 @@
+import os
+
+from ...utils import get_resources_dir
 from ...resources import Buffer, Texture
 from ._utils import to_vertex_format, to_texture_format, GfxSampler, GfxTextureView
 from ._shaderbase import BaseShader
 
 
+def load_shader(shader_name):
+    filename = os.path.join(get_resources_dir(), "shaders", shader_name)
+    with open(filename, "rb") as f:
```

Comment:
Maybe add encoding utf-8?

Reply:
oh wait, this reads as binary and then calls `.decode()` on the bytes (which is utf-8 by default).